File: api/src/models/leads.js
import Sequelize from 'sequelize';
import { sequelize } from '../database/database'
import User from './users';


const Leads =  sequelize.define('leads', {
    id:{
        type:Sequelize.INTEGER,
        primaryKey:true,
        autoIncrement: true,
    },
    campaignId:{
        type: Sequelize.STRING,
    },
    campaignName:{
        type: Sequelize.STRING,
    },
    email:{
        type: Sequelize.STRING
    },
    firstName:{
        type: Sequelize.STRING
    },
    lastName:{
        type: Sequelize.STRING
    },
    picture:{
        type: Sequelize.STRING
    },
    phone:{
        type: Sequelize.STRING,
        defaultValue:1
    },
    linkedinUrl:{
        type: Sequelize.STRING
    },
    companyName: {
        type: Sequelize.STRING
    },
    icebreaker:{
        type: Sequelize.STRING
    },
    Title:{
        type: Sequelize.STRING
    },
    Website:{
        type: Sequelize.STRING
    },
    Industry:{
        type: Sequelize.STRING
    },
    LinkedIn:{
        type: Sequelize.STRING
    },
    notes:{
        type: Sequelize.STRING
    },
    sentStep:{
        type: Sequelize.STRING
    },
    sentAt:{
        type: Sequelize.STRING,
        defaultValue: null
    },
    repliedAt:{
        type: Sequelize.STRING,
        defaultValue: null
    },
    openedAt:{
        type: Sequelize.STRING,
        defaultValue: null
    },
    bouncedAt:{
        type: Sequelize.STRING,
        defaultValue: null
    },
    unsubscribedAt:{
        type: Sequelize.STRING,
        defaultValue: null
    },
    clickedAt:{
        type: Sequelize.STRING,
        defaultValue: null
    },
    customer_id:{
        type: Sequelize.NUMBER,
        defaultValue: null
    },
    flag:{
        type: Sequelize.STRING,
        defaultValue: null
    }
});

export default Leads;

File: ReadMail/ReadMail/Models/Leads.cs
using System;
using System.Collections.Generic;

namespace ReadMail.Models
{
    public partial class Leads
    {
        public int Id { get; set; }
        public string CampaignId { get; set; }
        public string CampaignName { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Picture { get; set; }
        public string Phone { get; set; }
        public string LinkedinUrl { get; set; }
        public string CompanyName { get; set; }
        public string Icebreaker { get; set; }
        public string Title { get; set; }
        public string Website { get; set; }
        public string Industry { get; set; }
        public string LinkedIn { get; set; }
        public string Notes { get; set; }
        public string SentStep { get; set; }
        public string SentAt { get; set; }
        public string RepliedAt { get; set; }
        public string OpenedAt { get; set; }
        public string BouncedAt { get; set; }
        public string UnsubscribedAt { get; set; }
        public string ClickedAt { get; set; }
        public int? CustomerId { get; set; }
        public string Flag { get; set; }
    }
}
